types/testingutils/qbft_msgs.go

```diff
@@ -6,6 +6,18 @@ import (
 	"github.com/herumi/bls-eth-go-binary/bls"
 )
 
+var DefaultIdentifier = []byte{1, 2, 3, 4}
+var WrongRoot = [32]byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 1, 2, 3, 4, 5, 6, 7, 8, 9, 5, 6, 7, 8, 9}
+
+var MarshalJustifications = func(msgs []*qbft.SignedMessage) [][]byte {
+	bytes, err := qbft.MarshalJustifications(msgs)
+	if err != nil {
+		panic(err)
```

Comment:
it would be cooler to use the require package here

Reply:
@aaron-blox Yes, but right now we don't have `*testing.T` in tests that call `MarshalJustifications`